index.js

```diff
@@ -8,6 +11,10 @@
 
 
 // ***** Question 2 *****
+
+const name = "Patrick"
```

Comment:
Great! The key here was using `const` because it doesn't allow us to reassign our variable